sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -200,7 +202,8 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
           logInfo("Skip tests because old Spark versions don't support Java 21.")
         }
       } else {
-        logError(s"Python version <  ${TestUtils.minimumPythonSupportedVersion}, " +
+        logError(
+          log"Python version < ${MDC(VERSION, TestUtils.minimumPythonSupportedVersion)}, " +
           "the running environment is unavailable.")
```

Comment:
@itholic I guess you need to write it this way because `MDC` + `String` is not currently supported, 😄
"the running environment is unavailable." -> log"the running environment is unavailable."

Reply:
Oh, thanks for the correction!